src/hostname/hostnamed.c

```diff
@@ -18,13 +18,15 @@
 #include "fileio.h"
 #include "hostname-util.h"
 #include "id128-util.h"
+#include "libudev.h"
```

Comment:
Gah!

Reply:
oops! true! i should have noticed. wasn't awake properly i guess! sorry!

Reply:
np. I will prep a patch.

Reply:
#18019.